bloch_sphere/animate_bloch.py

```diff
@@ -142,6 +143,22 @@ def apply_gate_list(self, gates, final_wait=True):
                     print(f'Error: Custom gate arguments contain invalid '
                           f'floats {gate}.')
                 continue
+
+            if re.match('r[x,y,z][,;]', gate[:3].lower()):
+                try:
+                    r_pi = float(gate[3:])
+                except ValueError:
+                    print('Error: Rx/Ry/Rz gate should have style like '
+                          'Rx;{float} or Rx,{float}.')
+                gate_name = gate[:3].lower()
+                x = int(gate_name[1] == 'x')
+                y = int(gate_name[1] == 'y')
+                z = int(gate_name[1] == 'z') 
+                formated_r_pi = f"{r_pi:.3f}".rstrip('0').rstrip('.')
+                label = f"{gate_name}({formated_r_pi})"
```

Comment:
Do you still want the label to be capitalized like Rx?

Reply:
Personally, I think Rx looks better than rx. This change will make it print 'rx'. If you don't object, I will update it to Rx instead.

Reply:
Done. Glad to check it again (The previous code had a mistake that printing message includes ;)